### DISCUSSION TITLE
Init some data when collection first created

### DISCUSSION THREAD

**Discussion by ozil-cao** [isar/isar#1635] [Help]
Thanks for the project.I want to put some records when collection first created,after read the document,I still have't a good solution,do you have any good ideas?

**Comment by Jtplouffe** [comment]
There is a lot of ways this could be achieved, for example, you could have a key/value collection in which you could insert an entry like `{ "key": "collection_a_seeded", "value": true }`, and lookup to see if your collection was already seeded.

Another way, assuming that this collection should mostly never be empty after initial seeding, you could check the object count from the collection, and if `0`, initialize the seeding logic.
